CHANGELOG_PENDING.md

```diff
@@ -8,6 +8,7 @@ BREAKING CHANGES:
 - [p2p] Remove salsa and ripemd primitives, in favor of using chacha as a stream cipher, and hkdf
 - [abci] Changed time format from int64 to google.protobuf.Timestamp
 - [abci] Changed Validators to LastCommitInfo in RequestBeginBlock
+- [crypto] Switch crypto.Signature from interface to []byte for space efficiency
```

Comment:
Note to myself: investigate if this means we need to be update the kms code as well (doesn't seem so as we already just treat signatures as bytes there) /cc @zmanian